script/openqa-clone-custom-git-refspec

```diff
@@ -55,6 +53,23 @@ extract_urls_from_pr() {
     fi
 }
 
+config_file="$HOME/.config/openqa/github-token.conf"
+get_token_from_file() {
+    token=$(grep "^GITHUB_TOKEN=" "$config_file" | cut -d'"' -f2)
+    echo "$token"
```

Comment:
This should be equivalent, right?
```suggestion
    grep "^GITHUB_TOKEN=" "$config_file" | cut -d'"' -f2
```

Reply:
Not sure about if its equivalent, but I personally prefer a explicit return to make the code more expressive. As long as this is not breaking things I would prefer to keep it

Reply:
I don't think it makes sense to first assign to a variable and then expand the variable content within a string that only contains the variable and then output that. So I would also prefer the variant that @Martchus proposed

Reply:
Applied it 👍🏼